src/style.css

```diff
@@ -1 +1,57 @@
 /* add styles here */
+html {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
Don't use tag names for styling (except `html` and `body`). You can remove these styles as they are typically default for browsers, or if needed, apply them to the `body` tag instead.